### DISCUSSION TITLE
Would you like to join or provide feedback to the PowerShell Working Groups?

### DISCUSSION THREAD

**Discussion by kilasuit** [PowerShell/PowerShell#20823] [General]
Going forward we've identified a need to both expand the working groups and also make it easier to interact with them going forward.

Each Working Group could & I think should in future have their own community calls, which should be lead by the community more than the whole of the group which would then free up some time for the working groups to keep on with their workloads whilst not forcing them to be in yet another call unless they can/chose to attend. 

If you would like to see WG community calls then please react with  👀 to this & comment which WG's that you would like to interact with closer in future.
If you would like to be considered for joining the WG's then please add a comment to this [existing discussion](https://github.com/orgs/PowerShell/discussions/16501) for consideration

**Comment by dkattan** [comment]
I’d like to be in wg-engine

**Comment by Ayanmullick** [comment]
@kilasuit,  I would like to participate in community calls for `Developer Experience,` `Interactive UX`, and the `Language` WGs.
